src/scripting/Scripting.cpp

```diff
@@ -97,13 +97,44 @@ void Scripting::Initialize()
 
     luaGlobal["ModArchiveExists"] = [this](const std::string& acArchiveName) -> bool
     {
+        bool archiveExists = false;
```

Comment:
Update PR to latest master and use GetLuaPath instead of manually managing paths like this ideally.

Reply:
no longer valid due to RED4ext::ResourceDepot implementation